models/user.dao.js

```diff
@@ -0,0 +1,52 @@
+const { AppDataSource } = require("./dataSource");
+
+  const createuser = async (email, password, firstname, lastname, skintype) => {
+    
+    try{
+   
+    return AppDataSource.query(
+      `
+      INSERT INTO users (
+      email,
+      password, 
+      first_name,
+      last_name,
+      skin_type
+    ) VALUES (
+      ?,
+      ?,
+      ?,
+      ?,
+      ?
+    )
+    `, 
+    [email, password, firstname, lastname, skintype]
+    );
+    }  catch(err) {
+    const error= new Error('INVAID_DATA_INPUT');
+    error.statusCode = 500;
+    throw error;
+  }
+  
+};
+
+
+const connectuser = async (email) => {
+  const [user] = await AppDataSource.query(
+    `
+    SELECT 
+      u.id
```

Comment:
유저의 id 정보만 파악하면 email 혹은 password에 대한 정보를 확인할 수 없을 것 같습니다.
다른 컬럼에 대한 데이터도 받아올 수 있도록 수정해주세요.

```suggestion
      u.id,
      u.email,
      u.password
```